src/components/views/elements/MemberEventListSummary.js

```diff
@@ -437,7 +438,7 @@ module.exports = React.createClass({
         );
 
         return (
-            <div className="mx_MemberEventListSummary">
+            <div className="mx_MemberEventListSummary" data-contained-scroll-tokens={eventIds}>
```

Comment:
Even when expanded, `_scrollToToken` will only look through the direct children of the `ScrollPanel`, not the children of the MELS (which is itself a child of the panel).